seqr/views/apis/case_review_api_tests.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+import json
+
+from django.core.files.uploadedfile import SimpleUploadedFile
```

Comment:
We have an automated code quality checker called codacy. The integration with github is not working, but it flagged this line as an unused import: https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5244461